release/sdk-config.yaml.tmpl

```diff
@@ -46,12 +46,12 @@ commands:
   completion: true
 - name: sandbox
   path: daml-helper/daml-helper
-  desc: "Launch the Sandbox"
+  desc: "Launch sandbox"
```

Comment:
[We capitalize "Sandbox" in the docs](https://docs.daml.com/tools/sandbox.html), so I'd prefer to do that here too.

```suggestion
  desc: "Launch Sandbox"
```

Reply:
Ok, done!